modules/silauth/lib/Auth/Source/models/FailedLoginUsername.php

```diff
@@ -36,7 +36,7 @@ public function behaviors()
         ];
     }
     
-    public static function countRecentFailedLoginsFor($username)
+    public static function countRecentFailedLoginsFor(string $username): bool|int|string|null
```

Comment:
PhpStorm insisted on it. I think because of this line:
```
return $builder->createCountCommand($this->getTableSchema(),$criteria)->queryScalar();
```

Reply:
But it's probably better to deal with the unwanted return types and restrict it to just string.